modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -3189,14 +3191,17 @@ public void addLast(T cell) {
             }
         }
 
+        @Override
```

Comment:
The added `@Override` annotations are unrelated to your fix. Can you please revert them?

Reply:
By trying to make eclipse remove trailing whitespace I've accidentally activated the "Add missing Annotations" option :-(.
Will remove them and update the PR ...